packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -59,6 +64,22 @@ function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownPr
   const resetOptions =
     selectedDeviceSession?.deviceInfo.platform === "iOS" ? resetOptionsIOS : resetOptionsAndroid;
 
+  const rotateOptions: Array<{
+    label: string;
+    value: DeviceRotationType;
+    icon: string;
+    rotation: string;
+  }> = [
+    { label: "Portrait", value: "Portrait", icon: "device-mobile", rotation: "0deg" },
+    { label: "Landscape Left", value: "LandscapeLeft", icon: "device-mobile", rotation: "-90deg" },
+    {
+      label: "Landscape Right",
+      value: "LandscapeRight",
+      icon: "device-mobile",
+      rotation: "90deg",
+    },
+  ];
```

Comment:
NIT: I'd move this outside of the component:
1. so that you don't create new objects each time the component renders
2. that's how the other options here are handled
3. I personally think it's just nicer that way